modules/loans/src/lib.rs

```diff
@@ -43,7 +43,7 @@ pub trait Trait: system::Trait {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as Loans {
-		pub Debits get(fn debits): double_map hasher(blake2_256) T::AccountId, hasher(blake2_256) CurrencyIdOf<T> => T::DebitBalance;
+		pub Debits get(fn debits): double_map hasher(blake2_256) CurrencyIdOf<T>, hasher(blake2_256) T::AccountId => (T::DebitBalance, Option<(CurrencyIdOf<T>, T::AccountId)>);
```

Comment:
I want to do some changes upstream to make iterator returns keys as well. so we can use blake2_128_concat hasher and getting keys as well.

Also this needs migration if we want to runtime upgrade Mandala. But after we change the key hasher so we don't need to migrate twice.